frontend/src/components/viewers/Tensorboard.tsx

```diff
@@ -70,51 +89,167 @@ class TensorboardViewer extends Viewer<TensorboardViewerProps, TensorboardViewer
       <div>
         {this.state.podAddress && (
           <div>
-            <div className={padding(20, 'b')}>Tensorboard is running for this output.</div>
+            <div className={padding(20, 'b')}>{`Tensorboard is running for this output.`}</div>
             <a
               href={'apis/v1beta1/_proxy/' + podAddress}
               target='_blank'
               className={commonCss.unstyled}
             >
-              <Button className={commonCss.buttonAction} disabled={this.state.busy}>
+              <Button
+                className={commonCss.buttonAction}
+                disabled={this.state.busy}
+                style={{ marginBottom: 20, width: 150 }}
```

Comment:
Inline styles should only be used when it depends on dynamic data.
You can use typestyle for this, reference: https://github.com/kubeflow/pipelines/blob/1285fd9fd9a9b2caa4143b169f52e2190c38c7ab/frontend/src/components/Banner.tsx#L31

Reply:
There are other usages. Please change them too.

Reply:
OK. Will do.